pkg/apimachinery/identity/wrapper.go

```diff
@@ -13,6 +14,38 @@ type IDClaimsWrapper struct {
 	Source Requester
 }
 
+func (i *IDClaimsWrapper) NamespaceMatches(namespace string) bool {
```

Comment:
Not sure this logic needs to remain here. There is also namespace access checker in authlib, whose CheckAccess func contains similar logic in my branch. I developed the logic here but found a better home for it in authlib.

Reply:
There are no consumers for this func, just needed an implementation after authlib upgrade.